web/src/components/Download.vue

```diff
@@ -31,7 +31,7 @@ export default {
     dataset() { return this.$store.getters.dataset(this.id); },
     dataframe() { return this.dataset.validatedMeasurements; },
     ready() { return this.$store.getters.ready(this.id); },
-    loading() { return this.$store.state.loading; },
+    loading() { return this.$store.getters.persisting; },
```

Comment:
I agree, I didn't change it for now, since it is used in different areas of the pug templates and also in the plot data. Making sure that nothing breaks without a good refactoring tool was too much effort for me.